firebase-crashlytics/src/main/java/com/google/firebase/crashlytics/internal/model/CrashlyticsReport.java

```diff
@@ -1352,6 +1362,9 @@ public abstract static class Builder {
     @NonNull
     public abstract Builder setFirebaseInstallationId(@Nullable String value);
 
+    @NonNull
+    public abstract Builder setFirebaseAuthenticationToken(@Nullable String value);
```

Comment:
Wonder for for deserialization (Report to json), if authenticationToken set to `null`, are we passing `null` or `""`? For iOS we are passing empty string all the time for invalid states. Not sure if this is a problem for Android? Reference: https://github.com/firebase/firebase-ios-sdk/pull/12383

Reply:
Null means the field will not be included in the json serialization, so it won't waste data over the wire. Then the reader will read the "missing" field as a default value, which for a string, will be empty string.